test/cpp/util/grpc_tool.cc

```diff
@@ -527,6 +530,10 @@ bool GrpcTool::CallMethod(int argc, const char** argv,
     }
   }
 
+  if (FLAGS_timeout > 0) {
+    deadline.reset(new std::chrono::system_clock::time_point(std::chrono::system_clock::now() + std::chrono::seconds(FLAGS_timeout)));
```

Comment:
Is there a reason deadline is reset to now() + timeout, instead of now() being added when setting the deadline for the ClientContext in cli_call.cc?

Reply:
I wanted the `CliCall` constructor parameters to be consistent with gRPC terminology, so I passed a `deadline` parameter instead of `timeout`, and because of that `deadline` has to be constructed before `CliCall`.